content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. This makes the bugzilla plugin run a couple
+of checks:
+* If a bug with the id `123` exists and is in state `open`
+* If the target release of the bugzilla bug matches the configured target release for the branch the PR points at. This mapping is
+	configured in the [openshift/release repository][0] via the `target_release` property
+* If the bug is in the correct state. Accepted states are configured in the [openshift/release repository][0] through the `valid_states`
+  property
+* If the bug has `dependant_bugs` and `dependent_bug_states` configured, those dependant bugs must be in one of the configured states
```

Comment:
```suggestion
* If the bug has `dependent_bugs` and `dependent_bug_states` configured, those dependent bugs must be in one of the configured states
```
also do you mean if the repository has dependent_bugs configured?

Reply:
Interesting, `dependant` is only correct in Britisch English, TILd

Reply:
huh.  I didn't realize it was a british spelling.